lib/joi/joi.bundle.js

```diff
@@ -1630,7 +1630,7 @@ module.exports = internals.Any = class {
         }
```

Comment:
Automatically generated by Webpack according to the instructions from the dependency [`README`](https://github.com/OldSneerJaw/couchster/blob/115cf53e42d1b860224f0d04300c5a209fad311f/lib/joi/README.md).